core/src/main/java/com/novoda/utils/Log.java

```diff
@@ -0,0 +1,136 @@
+package com.novoda.utils;
+
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public abstract class Log {
```

Comment:
we should rename this so that clients of the library don't get spammed with `Log` classes 🍡 `NoPlayerLog`?